packages/frontend/src/js/app/components/map/map.js

```diff
@@ -304,8 +311,10 @@ export default {
 
     watch: {
         towns() {
-            if (this.loadTerritoryLayers) {
-                this.countNumberOfTowns();
+            this.countNumberOfTowns();
```

Comment:
Il ne faut pas faire `this.countNumberOfTowns()` si loadTerritoryLayers n'est pas à `true`. C'est indispensable pour ne pas causer un overload de 3/4 secondes à chaque chargement de la fiche site qui n'a pas besoin de toutes ces données !

Je pense que ta fonction devrait être :
```
if (this.loadTerritoryLayers) {
    this.countNumberOfTowns();
    this.loadRegionalData();
    this.loadDepartementalData();
    this.loadCityData();

    if (this.displayShantytownsLevel) {
        this.syncTownMarkers();
    }
}
```

Reply:
C'est corrigé. Pas d'effet de bord constaté.